modules/sdk-core/package.json

```diff
@@ -71,7 +71,9 @@
     "strip-hex-prefix": "^1.0.0",
     "superagent": "^9.0.1",
     "tweetnacl": "^1.0.3",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
+    "macaroon": "^3.0.4",
```

Comment:
Seems not a very well maintained library, Last publish is 6 years ago.

Reply:
Any other possible alternatives?

Reply:
Unfortunately no. Other alternative libs also has similar maintenance. This has most downloads. Writing macaroon functions are very low level.

BTW, reason for low maintenance- macaroon tokens are not popular and its like JWT, nothing much to improve.